[0m06:33:25.817449 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x100b7c5e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103c29d00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103c29ee0>]}


============================== 06:33:25.819630 | e16aaa38-afb4-4eeb-b9f1-adbe125fd890 ==============================
[0m06:33:25.819630 [info ] [MainThread]: Running with dbt=1.5.3
[0m06:33:25.819957 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/henri_nguemaleu/.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'send_anonymous_usage_stats': 'True'}
[0m06:33:25.820235 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/Users/henri_nguemaleu/.dbt')}
[0m06:33:25.820415 [info ] [MainThread]: Creating dbt configuration folder at 
[0m06:33:25.821129 [debug] [MainThread]: Starter project path: /Users/henri_nguemaleu/Library/Python/3.9/lib/python/site-packages/dbt/include/starter_project
[0m06:33:29.844048 [error] [MainThread]: Encountered an error:

[0m06:33:29.858061 [error] [MainThread]: Traceback (most recent call last):
  File "/Users/henri_nguemaleu/Library/Python/3.9/lib/python/site-packages/dbt/cli/requires.py", line 86, in wrapper
    result, success = func(*args, **kwargs)
  File "/Users/henri_nguemaleu/Library/Python/3.9/lib/python/site-packages/dbt/cli/requires.py", line 71, in wrapper
    return func(*args, **kwargs)
  File "/Users/henri_nguemaleu/Library/Python/3.9/lib/python/site-packages/dbt/cli/main.py", line 459, in init
    results = task.run()
  File "/Users/henri_nguemaleu/Library/Python/3.9/lib/python/site-packages/dbt/task/init.py", line 306, in run
    adapter = self.ask_for_adapter_choice()
  File "/Users/henri_nguemaleu/Library/Python/3.9/lib/python/site-packages/dbt/task/init.py", line 230, in ask_for_adapter_choice
    numeric_choice = click.prompt(prompt_msg, type=click.INT)
  File "/Users/henri_nguemaleu/Library/Python/3.9/lib/python/site-packages/click/termui.py", line 166, in prompt
    value = prompt_func(prompt)
  File "/Users/henri_nguemaleu/Library/Python/3.9/lib/python/site-packages/click/termui.py", line 149, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m06:33:29.859011 [debug] [MainThread]: Command `dbt init` failed at 06:33:29.858867 after 4.05 seconds
[0m06:33:29.859382 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x100b7c5e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103c50700>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x103c506a0>]}
[0m06:33:29.859732 [debug] [MainThread]: Flushing usage events
[0m06:34:35.784976 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104d345e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107de1d90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107de1f70>]}


============================== 06:34:35.787220 | 0ffc263b-2f23-4f57-9a47-73cdc7e90660 ==============================
[0m06:34:35.787220 [info ] [MainThread]: Running with dbt=1.5.3
[0m06:34:35.787544 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/Users/henri_nguemaleu/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'send_anonymous_usage_stats': 'True'}
[0m06:34:35.788149 [info ] [MainThread]: A project called dbtlearn already exists here.
[0m06:34:35.788497 [debug] [MainThread]: Command `dbt init` succeeded at 06:34:35.788428 after 0.01 seconds
[0m06:34:35.788658 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104d345e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104d341f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107de1d90>]}
[0m06:34:35.788816 [debug] [MainThread]: Flushing usage events
[0m06:35:10.809733 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1010405e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1040edd90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1040edf70>]}


============================== 06:35:10.812082 | 8cc6814b-2643-406e-b229-1f5d082edc81 ==============================
[0m06:35:10.812082 [info ] [MainThread]: Running with dbt=1.5.3
[0m06:35:10.812441 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/henri_nguemaleu/.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'static_parser': 'True', 'log_format': 'default', 'target_path': 'None', 'send_anonymous_usage_stats': 'True'}
[0m06:35:10.813082 [debug] [MainThread]: Starter project path: /Users/henri_nguemaleu/Library/Python/3.9/lib/python/site-packages/dbt/include/starter_project
[0m06:39:32.426435 [info ] [MainThread]: Profile dbtlearning written to /Users/henri_nguemaleu/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m06:39:32.430184 [info ] [MainThread]: 
Your new dbt project "dbtlearning" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m06:39:32.432534 [debug] [MainThread]: Command `dbt init` succeeded at 06:39:32.432096 after 261.65 seconds
[0m06:39:32.433602 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1010405e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1326798e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1326795b0>]}
[0m06:39:32.434747 [debug] [MainThread]: Flushing usage events
[0m13:54:04.840147 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106f3e850>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x120185df0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1201cc4f0>]}


============================== 13:54:04.842580 | 8d210ab3-896c-40da-b5a9-c1e84b4f8d2d ==============================
[0m13:54:04.842580 [info ] [MainThread]: Running with dbt=1.5.3
[0m13:54:04.842894 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/henri_nguemaleu/.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'send_anonymous_usage_stats': 'True'}
[0m13:54:04.843197 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/henri_nguemaleu/Documents/Learning/DBT/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m13:54:04.843563 [debug] [MainThread]: Command `dbt run` failed at 13:54:04.843498 after 0.02 seconds
[0m13:54:04.843734 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106f3e850>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x120185100>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x120185bb0>]}
[0m13:54:04.843900 [debug] [MainThread]: Flushing usage events
